charts/incubator/teslamate/questions.yaml

```diff
@@ -10,50 +10,50 @@ questions:
 # Include{replicas1}
 # Include{podSpec}
 # Include{containerMain}
-                                - variable: env
-                                  label: "Image Environment"
-                                  schema:
-                                    additional_attrs: true
-                                    type: dict
-                                    attrs:
-                                      - variable: DISABLE_MQTT
-                                        label: "Mqtt Disabled"
-                                        description: "Disables the MQTT feature if true"
-                                        schema:
-                                          type: boolean
-                                          default: true
-                                      - variable: MQTT_HOST
-                                        label: "Mqtt Host"
-                                        description: "Hostname of the broker."
-                                        schema:
-                                          type: string
-                                          default: ""
-                                          show_if: [["DISABLE_MQTT", "=", false]]
-                                      - variable: MQTT_PORT
-                                        label: "Mqtt Port"
-                                        schema:
-                                          type: int
-                                          default: 1883
-                                          show_if: [["DISABLE_MQTT", "=", false]]
-                                      - variable: MQTT_USERNAME
-                                        label: "Mqtt Username"
-                                        description: "Username of the broker."
-                                        schema:
-                                          type: string
-                                          default: ""
-                                          show_if: [["DISABLE_MQTT", "=", false]]
-                                      - variable: MQTT_PASSWORD
-                                        label: "Mqtt Password"
-                                        description: "Password of the broker."
-                                        schema:
-                                          type: string
-                                          private: true
-                                          show_if: [["DISABLE_MQTT", "=", false]]
-
 # Include{containerBasic}
 # Include{containerAdvanced}
 # Include{containerConfig}
 # Include{podOptions}
+  - variable: env
```

Comment:
Why is this moved here? This is not correct.

Reply:
I totally missed that it is under the wrong category while reviewing it.

Reply:
Indentation is also wrong. It was fine before

Reply:
Yeah I thought so aswell. But the automatic testing didn't pick it up.
Should I revert the merge or just make a new pr and fix the issue I missed while reviewing?

Reply:
New pr is fine 

ci only tests validity of questions formatting. Not the structure.

Ci only tests based on values

Reply:
it was already broken anyway, no harm in leaving it broken. I can make a follow up PR.

Reply:
I based it off authelia: https://github.com/truecharts/charts/blob/master/charts/premium/authelia/questions.yaml#L16

is this something different?

Reply:
Env is a reserved section. In a specific place

Reply:
got it, I will make a new PR now